connectors/cli.py

```diff
@@ -44,11 +44,20 @@ def _parser():
         default=os.path.join(os.path.dirname(__file__), "..", "config.yml"),
     )
 
-    parser.add_argument(
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
+        "--log-level",
+        type=str,
+        choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
+        default=None,
```

Comment:
shouldn't we have a default that one of the `choices` items?

Reply:
I think Artem want to use the log level in config if it's not specified in args.

Reply:
Yeah if we default to something here, the setting from config file is never picked up